docs/jwst/outlier_detection/arguments.rst

```diff
@@ -1,42 +1,42 @@
 .. _outlier_detection_step_args:
 
+For more details about step arguments (including datatypes, possible values
+and defaults) see :py:obj:`jwst.outlier_detection.OutlierDetectionStep.spec`.
+
 Step Arguments for Non-IFU data
 ===============================
 The `outlier_detection` step for non-IFU data has the following optional arguments
 that control the behavior of the processing:
 
-``--weight_type`` (string, default='exptime')
-  The type of data weighting to use during resampling;
-  options are 'exptime', 'error', and 'None'.
+``--weight_type``
+  The type of data weighting to use during resampling.
 
-``--pixfrac`` (float, default=1.0)
-  The pixel fraction used during resampling;
-  valid values go from 0.0 to 1.0.
+``--pixfrac``
+  The pixel fraction used during resampling.
 
-``--kernel`` (string, default='square')
+``--kernel``
   The form of the kernel function used to distribute flux onto a
-  resampled image. Options are 'square', 'turbo', 'point', and
-  'lanczos'.
+  resampled image.
 
-``--fillval`` (string, default='INDEF')
+``--fillval``
   The value to assign to resampled image pixels that have zero weight or
   do not receive any flux from any input pixels during drizzling.
   Any floating-point value, given as a string, is valid.
   A value of 'INDEF' will use the last zero weight flux.
 
-``--nlow`` (integer, default=0)
-  The number of low values in each pixel stack to ignore
-  when computing the median value.
+``--nlow``
+  Deprecated and has no effect. This parameter will be removed
+  in a future version.
 
-``--nhigh`` (integer, default=0)
-  The number of high values in each pixel stack to ignore
-  when computing the median value.
+``--nhigh``
+  Deprecated and has no effect. This parameter will be removed
+  in a future version.
 
-``--maskpt`` (float, default=0.7)
+``--maskpt``
   The percent of maximum weight to use as lower-limit for valid data;
   valid values go from 0.0 to 1.0.
 
-``--snr`` (string, default='4.0 3.0')
+``--snr``
   The signal-to-noise values to use for bad pixel identification.
```

Comment:
Given that the default value is being removed and hence the reader no longer sees an example of how the 2 values are specified, I think the description here should be updated to include an example, e.g. '4.0 3.0'.